editor/src/components/editor/store/dispatch-strategies.tsx

```diff
@@ -164,6 +165,7 @@ export function interactionUpdate(
   strategies: Array<CanvasStrategy>,
   storedState: EditorStoreFull,
   result: InnerDispatchResult,
+  isInteractionAction: boolean,
```

Comment:
this boolean flag is pretty hard to read in the tests. one thing we'be been doing in some places is to use a string literal union instead of a boolean, so reading the callsite explains what the flag does.